src/docs/patterns/appbar.mdx

```diff
@@ -35,3 +37,37 @@ In some situations, you may want to present more information in an app bar, such
 <DemoCard repository={'search-bar'} angular react={'https://codesandbox.io/s/github/pxblue/react-design-patterns/tree/master?file=/src/pages/app-bar/search-bar/index.tsx&initialpath=search'} ionic reactNative />
 
 A very common pattern seen in many types of applications is the ability to search for data on a page. This is achieved by placing a search icon/button in the app bar. When this button is clicked, a white Search Bar slides in from the right to cover the default toolbar. As the user enters text into the search field, the data is filtered in real-time.
+
+<Divider />
+
+# Design Specifications
+
+For additional instructions, refer to [Material Design's design specs on "App bars: top"](https://material.io/components/app-bars-top#specs).
+
+## Mobile
+
+<ImageGrid images={[Specs1Image]} regularWidth />
+
+## Desktop and Tablet
+
+If certain dimensions are not specified, refer to the dimensions suggested in the mobile section above.
```

Comment:
There are no dimensions suggested in the mobile section above

Reply:
It works like this: 
- look at desktop & tablet dimension
- has no dimension 
- look at mobile
- has no mobile dimension
- look at the Material Design spec
- Aha, here it is

So kinda like some inherit thing... or should we just "import" the material dimensions, so that people don't have to look here and there?

Reply:
It might be simplest to just include the material dimensions instead of having to go through the path of inheritance.